BUILD

```diff
@@ -36,9 +36,9 @@ go_test(
 #### Atlassian / Localstack ####
 genrule(
     name = "dynamodb",
-    outs = ["localstack-run-id"],
-    cmd = "docker stop localstack || true; docker rm -f localstack || true; docker run -d -p 4567-4576:4567-4576 --name localstack atlassianlabs/localstack:0.4.1 > $@",
+    outs = ["dynamodb-local-run-id"],
+    cmd = "docker stop dynamodb-local || true; docker rm -f dynamodb-local || true; docker run -d -p 4569:8000 --name dynamodb-local amazon/dynamodb-local:latest > $@",
```

Comment:
👍 I trust that this image is better than the localstack (dynalite) one?

Reply:
I think this is "officially" provided by Amazon so I would hope so.